tikv-client/src/main/java/com/pingcap/tikv/meta/TiDAGRequest.java

```diff
@@ -281,31 +296,6 @@ private DAGRequest buildTableScan() {
     return request;
   }
 
-  private void setColumnOffsets(TiExpr expr) {
```

Comment:
Same as above, please refer to https://github.com/pingcap/tikv-client-lib-java/pull/203